Gruntfile.js

```diff
@@ -67,7 +72,7 @@ module.exports = function (grunt) {
         watch: {
         templates: {
                 files: 'modules/**/*.html',
-                tasks: 'ngtemplates'
+                tasks: ['build', 'eslint']
```

Comment:
I think `ngtemplates` should be the only task here, but then add another watcher to watch the source files (src js and templates.js) and run `build`. Also we shouldn't be linting `templates.js` because it's auto-compiled and as such we can't control it very well